### ISSUE TITLE
`reportDuplicateImport` false negative when duplicate import is in separate import statements

### ISSUE THREAD

**Issue by DetachHead** [microsoft/pyright#7261]
pyright correctly reports `reportDuplicateImport` if the duplicated import is part of the same import statement:
```py
from typing import cast, cast # error: "cast" is imported more than once
```

but not when they are in separate import statements:
```py
from typing import cast
from typing import cast # no error
```

[playground](https://pyright-play.net/?reportDuplicateImport=true&code=GYJw9gtgBALgngBwJYDsDmUkQWEMoDGAhgM4wCwAUKJLIqhljnoafgMRQphQCmI4EEA)

**Comment by erictraut** [comment]
Pyright is working as designed here.

**Comment by DetachHead** [comment]
@erictraut how? It's literally a duplicated import that isn't being reported by the `reportDuplicateImport` rule
